oar/cli/cmd_change_advisory_status.py

```diff
@@ -30,6 +30,11 @@ def change_advisory_status(ctx, status):
         am = AdvisoryManager(cs)
         # update task status to in progress
         report.update_task_status(LABEL_TASK_CHANGE_AD_STATUS, TASK_STATUS_INPROGRESS)
+        # check kernel tag before change advisories' status
+        ads = am.get_advisories()
+        for ad in ads:
+            if Advisory(errata_id=ad.errata_id, impetus=ad.impetus).check_kernel_tag():
```

Comment:
ad is Advisory object, don't have to re-init, ref: https://github.com/openshift/release-tests/blob/master/oar/core/advisory.py#L39